server/rpc/httphealth/httphealth.go

```diff
@@ -24,46 +24,41 @@ import (
 	"connectrpc.com/grpchealth"
 )
 
-// serviceName is the path for the health check endpoint.
-const serviceName = "/healthz/"
-
 // CheckResponse represents the response structure for health checks.
 type CheckResponse struct {
 	Status string `json:"status"`
 }
 
 // NewHandler creates a new HTTP handler for health checks.
 func NewHandler(checker grpchealth.Checker) (string, http.Handler) {
+	const serviceName = "/yorkie.v1/healthz/"
```

Comment:
How about changing the service name to `/yorkie.v1.Health/Check` to correspond with `/grpc.health.v1.Health/Check` and `/yorkie.v1.YorkieService/XXX` to keep consistency with other RPCs?

Also, I think we don't need the ending `/` slash, since we are using `?` query parameter after.